src/find.ts

```diff
@@ -0,0 +1,84 @@
+import * as fs from "fs";
+import * as vscode from "vscode";
+import * as path from "path";
+import * as os_utils from './os-utils';
+
+
+export async function findAya(context: vscode.ExtensionContext): Promise<string | null> {
+  const config = vscode.workspace.getConfiguration("aya");
+
+  if (!config.get<boolean>("lsp.enabled")) {
+    await vscode.window.showInformationMessage("Aya language server is disabled");
+    return null;
+  }
+
+  let lspFatJarPath = config.get<string>("lsp.fatJar");
+  let lspExecPath = config.get<string>("lsp.exec");
+  if (lspFatJarPath && fs.existsSync(lspFatJarPath)) {
```

Comment:
Why sync?

Reply:
You can `await fsExists` I guess

Reply:
I think there's no need to be async when we only check file existence.

Reply:
Oh I see. We are traversing the file system. Then it's fine